packages/utils/src/object.ts

```diff
@@ -303,7 +303,7 @@ export function walk(key: string, value: any, depth: number = +Infinity, memo: M
   const acc = Array.isArray(value) ? [] : {};
 
   // If we already walked that branch, bail out, as it's circular reference
-  if (memo.memoize(value)) {
+  if (memo[0](value)) {
```

Comment:
taking the reacts hook approach to being minification friendly (return a tuple) 😄